src/DynamoCoreWpf/Views/SplashScreen/SplashScreen.xaml.cs

```diff
@@ -19,34 +23,183 @@ public partial class SplashScreen : Window
         private static readonly string backgroundImage = "Dynamo.Wpf.Views.SplashScreen.WebApp.splashScreenBackground.png";
         private static readonly string imageFileExtension = "png";
 
+        // Timer used for Splash Screen loading
         private Stopwatch loadingTimer;
-
         private long totalLoadingTime;
 
-        private DirectoryInfo webBrowserUserDataFolder;
-
         internal Action<bool> RequestLaunchDynamo;
         internal Action<string> RequestImportSettings;
         internal Func<bool> RequestSignIn; 
         internal Func<bool> RequestSignOut;
         internal WebView2 webView;
 
+        /// <summary>
+        /// Dynamo auth manager reference
+        /// </summary>
+        internal AuthenticationManager authManager;
+
+        /// <summary>
+        /// Dynamo View Model reference
+        /// </summary>
+        internal DynamoViewModel viewModel;
+
+        private DynamoView dynamoView;
+        /// <summary>
+        /// Dynamo View reference
+        /// </summary>
+        public DynamoView DynamoView
+        {
+            get
+            {
+                return dynamoView;
+            }
+            set
+            {
+                dynamoView = value;
```

Comment:
is this variable set in any place right now? or is just preparation for when the code is called from DynamoRevit?

Reply:
This setter is currently called in CoreSetupfile